lightning/src/ln/channelmanager.rs

```diff
@@ -1203,9 +1203,16 @@ pub enum PaymentSendFailure {
 	/// You can freely retry the payment in full (though you probably want to do so over different
```

Comment:
Maybe this should be `resend` then also to maintain the `retry` vs. `resend` difference?